packages/contentful/package.json

```diff
@@ -17,8 +17,8 @@
     "space:create-environment:crn": "TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\"}' ts-node ./scripts/create-environment.ts",
     "space:setup-webhook": "ts-node ./scripts/setup-webhook.ts",
     "space:remove-webhook": "ts-node ./scripts/remove-webhook.ts",
-    "schema:update:crn": "APP_NAME=crn graphql-codegen --config codegen.yml && yarn run prettier --write ./src/crn/autogenerated-gql/*",
-    "schema:update:gp2": "graphql-codegen --config codegen.yml && yarn run prettier --write ./src/gp2/autogenerated-gql/*"
+    "schema:update:crn": "APP=crn graphql-codegen --config codegen.yml && yarn run prettier --write ./src/crn/autogenerated-gql/*",
+    "schema:update:gp2": "APP=gp2 graphql-codegen --config codegen.yml && yarn run prettier --write ./src/gp2/autogenerated-gql/*"
```

Comment:
We won't need to do this though.

Reply:
No particular objection to it being removed, but while it exists it should work.